examples/Yolo-object-detection/client/model.py

```diff
@@ -0,0 +1,39 @@
+import os
+import numpy as np
+import subprocess
+
+from fedn.utils.helpers.helpers import get_helper
+
+HELPER_MODULE = "numpyhelper"
+helper = get_helper(HELPER_MODULE)
+dir_path = os.path.dirname(os.path.realpath(__file__))
+
+def git_clone(repo_url="https://github.com/AlexeyAB/darknet.git", clone_dir="../.darknet"):
```

Comment:
Why do we need to do a git clone "from code"? Is it not possible to install it as a library in the env?